README.md

```diff
@@ -137,13 +140,43 @@ For more information about the `remote_write` configuration that can be set with
 |-----------------------------------------------------|-------------------------------------------------------|-------------|
 | `grafana.enabled`                                   | If false, Grafana will not be created                 | `true`      |
 | `grafana.sidecar.datasources.enabled`               | If false, no data sources will be provisioned         | `true`      |
-| `grafana.sidecar.datasources.prometheus.enabled`    | If false, a Prometheus data source will not be provisioned | `true` |
-| `grafana.sidecar.datasources.prometheus.urlTemplate` | Template parsed to the url of the Prometheus API. Defaults to Prometheus deployed with this chart  | `http://{{ .Release.Name }}-prometheus-service.{{ .Release.Namespace }}.svc.cluster.local` |
-| `grafana.sidecar.datasources.timescaledb.enabled` | If false a TimescaleDB data source will not be provisioned | `true` |
-| `grafana.sidecar.datasources.timescaledb.user` | User to connect to TimescaleDB | `postgres`
-| `grafana.sidecar.datasources.timescaledb.urlTemplate` | Template that will be parsed to the host of TimescaleDB, defaults to DB deployed with this chart | `{{ .Release.Name }}.{{ .Release.Namespace }}.svc.cluster.local` |
 | `grafana.sidecar.dashboards.enabled`                | If false, no dashboards will be provisioned by default | `true`     |
 | `grafana.sidecar.dashboards.files`                  | Files with dashboard definitions (in JSON) to be provisioned | `['dashboards/k8s-cluster.json','dashboards/k8s-hardware.json']` |
+| `grafana.prometheus.ds.enabled` | If false, a Prometheus data source will not be provisioned | true |
```

Comment:
refactored the values for the grafana provisioned data sources, since some of the fields were repeatedly required for database and Timescale data source